### PR TITLE
[PM-13346] Email notification impacts

### PR DESCRIPTION
**Pull request by cyprain-okeke**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-13346

## 📔 Objective

Send email once policy is turned on to all members who have a free Families plan through the organization. This will only be sent to members who have activated the sponsorship.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by jonashendrickx — APPROVED** [review]

File: apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts [hunk 1]
```diff
@@ -29,7 +39,7 @@ export class AcceptFamilySponsorshipComponent extends BaseAcceptComponent {
     if (!qParams.register) {
       // FIXME: Verify that this floating promise is intentional. If it is, add an explanatory comment and ensure there is proper error handling.
```
Comment (r-tome) [inline apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts:40]: Its no longer a floating promise so these comments can be removed

**Review by r-tome — CHANGES_REQUESTED** [review]
It seems like there's some changes from https://github.com/bitwarden/clients/pull/11894 so change this PR to target that branch, please.

It would be great if you could add some screenshots.

File: apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts (hunk 1 above)
Comment (cyprain-okeke) [reply apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts]: removed

File: apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts [hunk 2]
```diff
@@ -19,6 +24,11 @@ export class AcceptFamilySponsorshipComponent extends BaseAcceptComponent {
...
   requiredParameters = ["email", "token"];
 
+  policyResponse!: OrganizationSponsorshipResponse;
+  policyApiService = inject(PolicyApiServiceAbstraction);
+  configService = inject(ConfigService);
+  toastService = inject(ToastService);
+
   async authedHandler(qParams: Params) {
     // FIXME: Verify that this floating promise is intentional. If it is, add an explanatory comment and ensure there is proper error handling.
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
```
Comment (r-tome) [inline apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts:34]: Are these still needed?

File: apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts [hunk 3]
```diff
@@ -27,9 +37,8 @@ export class AcceptFamilySponsorshipComponent extends BaseAcceptComponent {
 
   async unauthedHandler(qParams: Params) {
     if (!qParams.register) {
-      // FIXME: Verify that this floating promise is intentional. If it is, add an explanatory comment and ensure there is proper error handling.
       // eslint-disable-next-line @typescript-eslint/no-floating-promises
```
Comment (r-tome) [inline apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts:40]: This should be deleted too, I think

**Review by r-tome — APPROVED** [review]
I think there's a couple more comments to remove. Other than that I think it looks good!

File: apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts (hunk 3 above)
Comment (cyprain-okeke) [reply apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts]: removed

File: apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts (hunk 2 above)
Comment (cyprain-okeke) [reply apps/web/src/app/admin-console/organizations/sponsorships/accept-family-sponsorship.component.ts]: removed

**Pull request merged**